theme/static/js/hs_file_browser.js

```diff
@@ -620,8 +622,10 @@ function paste(destPath) {
 
     // Wait for the asynchronous call to finish to get new folder structure
     $.when.apply($, calls).done(function () {
-        refreshFileBrowser();
-        clearSourcePaths();
+        if (!is_async_celery_task) {
```

Comment:
I'm confused about how this works.  The comments a few lines up says `// Wait for the asynchronous call to finish to get new folder structure` but the folder structure is only refreshed when it is not an async task.

Reply:
Yeah, the asynchronous call in the comment meant the asynchronous called pushed to the calls stack in JavaScript which is different from what is_async_celery_task variable means. Will update the comment to make this clear.